e2e/TestConstants.ts

```diff
@@ -85,13 +85,47 @@ export const TestConstants = {
     TS_SELENIUM_HAPPY_PATH_WORKSPACE_NAME: process.env.TS_SELENIUM_HAPPY_PATH_WORKSPACE_NAME || 'petclinic-dev-environment',
 
     /**
-     * Username used to log in MultiUser Che
+     * Name of namespace created for 'Install Che' on OCP 4.x by OperatorHub UI.
+     */
+    TS_INSTALL_CHE_PROJECT_NAME: process.env.TS_INSTALL_CHE_PROJECT_NAME || 'test-che-operator',
+
+    /**
+     * Username used to log in MultiUser Che.
      */
     TS_SELENIUM_USERNAME: process.env.TS_SELENIUM_USERNAME || 'che',
 
     /**
-     * Password used to log in MultiUser Che
+     * Password used to log in MultiUser Che.
+     */
+    TS_SELENIUM_PASSWORD: process.env.TS_SELENIUM_PASSWORD || '',
+
+    /**
+     * Url of console OpenShift 4.x.
+     */
+    TS_SELENIUM_OPENSHIFT4_URL: process.env.TS_SELENIUM_OPENSHIFT4_URL || '',
+
+    /**
+     * Username used to log in OCP 4.x.
+     */
+    TS_SELENIUM_OPENSHIFT4_USERNAME: process.env.TS_SELENIUM_OPENSHIFT4_USERNAME || 'kubeadmin',
+
+    /**
+     * Password used to log in OCP 4.x.
+     */
+    TS_SELENIUM_OPENSHIFT4_PASSWORD: process.env.TS_SELENIUM_OPENSHIFT4_PASSWORD || '',
+
+    /**
+     * Log into CHE in MultiUser mode, "false" by default.
      */
-    TS_SELENIUM_PASSWORD: process.env.TS_SELENIUM_PASSWORD || ''
+    TS_SELENIUM_MULTIUSER: process.env.TS_SELENIUM_MULTIUSER === 'true',
```

Comment:
I am not sure about the default assembly. What is better for us "singleuser" or "multiuser"? @rhopp @musienko-maxim what do you think about that?

Reply:
Before the single user assembly was as default